urllib3/contrib/appengine.py

```diff
@@ -189,6 +189,10 @@ def _get_absolute_timeout(self, timeout):
                 warnings.warn(
                     "URLFetch does not support granular timeout settings, "
                     "reverting to total timeout.", AppEnginePlatformWarning)
+            if not timeout.total:
```

Comment:
sure! timeout 0 is generally either a special case or degenerate, but regardless, agreed. done.